vnext/ReactUWP/Utils/PropertyUtils.h

```diff
@@ -174,26 +174,47 @@ bool TryUpdateBorderProperties(
       UpdateControlBorderResourceBrushes(element, nullptr);
     }
   } else if (propertyName == "borderLeftWidth") {
-    if (propertyValue.isNumber())
+    if (propertyValue.isNumber()) {
       SetBorderThickness(node, element, ShadowEdges::Left, propertyValue.asDouble());
+    } else if (propertyValue.isNull()) {
+      SetBorderThickness(node, element, ShadowEdges::Left, 0);
```

Comment:
Nice catch.  Is there a test for this scenario?

Reply:
I have this image test to test three scenarios: no border, switch to border and then switch back to no border. The third scenario was where I found this cleanup code missing.

---
In reply to: [356349717](https://github.com/microsoft/react-native-windows/pull/3754#discussion_r356349717) [](ancestors = 356349717)